tap_suiteql/streams.py

```diff
@@ -124,86 +93,78 @@ class ChangeOrderLineStream(suiteqlStream):
-        th.Property("transactionnumber",th.StringType),
-        th.Property("type",th.StringType),
-        th.Property("typebaseddocumentnumber",th.StringType),
-        th.Property("userevenuearrangement",th.StringType),
-        th.Property("visibletocustomer",th.StringType),
-        th.Property("void",th.StringType),
-        th.Property("voided",th.StringType)
+        th.Property("abbrevtype", th.StringType),
+        th.Property("balsegstatus", th.StringType),
+        th.Property("billingstatus", th.StringType),
+        th.Property("closedate", th.DateTimeType),
+        th.Property("createdby", th.StringType),
+        th.Property("createddate", th.DateTimeType),
+        th.Property("currency", th.StringType),
+        th.Property("daysopen", th.StringType),
+        th.Property("entity", th.StringType),
+        th.Property("exchangerate", th.StringType),
+        th.Property("foreignpaymentamountunused", th.StringType),
+        th.Property("foreignpaymentamountused", th.StringType),
+        th.Property("foreigntotal", th.StringType),
+        th.Property("id", th.StringType),
+        th.Property("includeinforecast", th.StringType),
+        th.Property("isfinchrg", th.StringType),
+        th.Property("isreversal", th.StringType),
+        th.Property("lastmodifiedby", th.StringType),
+        th.Property("lastmodifieddate", th.DateTimeType),
+        th.Property("legacytax", th.StringType),
+        th.Property("number", th.StringType),
+        th.Property("onetime", th.StringType),
+        th.Property("ordpicked", th.StringType),
+        th.Property("paymenthold", th.StringType),
+        th.Property("posting", th.StringType),
+        th.Property("postingperiod", th.StringType),
+        th.Property("printedpickingticket", th.StringType),
+        th.Property("recordtype", th.StringType),
+        th.Property("recurannually", th.StringType),
+        th.Property("recurmonthly", th.StringType),
+        th.Property("recurquarterly", th.StringType),
+        th.Property("recurweekly", th.StringType),
+        th.Property("status", th.StringType),
+        th.Property("taxdetailsoverride", th.StringType),
+        th.Property("taxpointdateoverride", th.StringType),
+        th.Property("taxregoverride", th.StringType),
+        th.Property("trandate", th.DateTimeType),
+        th.Property("trandisplayname", th.StringType),
+        th.Property("tranid", th.StringType),
+        th.Property("transactionnumber", th.StringType),
+        th.Property("type", th.StringType),
+        th.Property("typebaseddocumentnumber", th.StringType),
+        th.Property("userevenuearrangement", th.StringType),
+        th.Property("visibletocustomer", th.StringType),
+        th.Property("void", th.StringType),
+        th.Property("voided", th.StringType),
     ).to_dict()
 
+
 class CustomlistGpyCompanysizeStream(suiteqlStream):
-    name = "CustomlistGpyCompanysize"
+    name = "customlist_gpy_companysize"
```

Comment:
Why we are changing the name in the custom streams?

Reply:
We are passing the name as the name of the entity to be used, in sql we have something like `select cols from <stream name>`. So for `CustomlistGpyCompanysizeStream` and `CustomlistGpyCompanysizeStream` we change the name of the stream.

Reply:
We use the stream name in the query

Reply:
I propose we create a attribute called entity_name. If we have an entity name we use that on the "from" statement otherwise we use the "name" parameter